src/console_func.h

```diff
@@ -43,7 +43,7 @@ inline void IConsolePrint(TextColour colour_code, const T &format, A first_arg,
 {
 	/* The separate first_arg argument is added to aid overloading.
 	 * Otherwise the calls that do no need formatting will still use this function. */
-	IConsolePrint(colour_code, fmt::format(format, first_arg, other_args...));
+	//IConsolePrint(colour_code, fmt::format(fmt::runtime(format), first_arg, other_args...));
```

Comment:
```suggestion
template <typename A, typename ... Args>
inline void IConsolePrint(TextColour colour_code, fmt::format_string<A, Args...> format, A first_arg, Args&&... other_args)
{
	/* The separate first_arg argument is added to aid overloading.
	 * Otherwise the calls that do no need formatting will still use this function. */
	IConsolePrint(colour_code, fmt::format(format, std::forward<A>(first_arg), std::forward<Args>(other_args)...));
}
```